pontoon/projects/management/commands/send_deadline_notifications.py

```diff
@@ -45,7 +45,12 @@ def handle(self, *args, **options):
             ).distinct()
 
             for contributor in contributors:
-                notify.send(project, recipient=contributor, verb=verb)
+                if (
+                    Project.objects.filter(pk=project.pk)
```

Comment:
This will run a DB query for each user... Maybe we can instead tweak the `contributors` QuerySet above with the same trick as here https://github.com/mozilla/pontoon/pull/1619/commits/c864d025b712313f4efc83b5931f31b4d3305450#diff-c18ce63ddb8aca155c0553a0c1b174e3?

Reply:
I can optimize this, but I don't think there's an easy way to re-use `visibility_for` in this context. I can create a separate function, similar to this on -> https://github.com/mozilla/pontoon/pull/1619/files/edbd0672ab405c8f64158821eb24c6276323f627#diff-cfad39fede62fdc24a97edc6622b538dR546